### PR TITLE
RF_03.001.02| Multi-configuration project > Edit description on the status page > POM

### PR DESCRIPTION
**Pull request by KuperWoman**

### CODE REVIEW

**Comment by KuperWoman** [comment]
[RF_03.001.02| Multi-configuration project > Edit description on the status page > POM](https://trello.com/c/C9xdKg1t/923-rf0300102-multi-configuration-project-edit-description-on-the-status-page-pom)

**Review by Gaztechica — APPROVED** [review]

File: src/test/java/school/redrover/MultiConfigurationProjectTest.java [hunk 1]
```diff
@@ -59,26 +59,24 @@ public void testAddDescription() {
...
-        getWait5().until(ExpectedConditions.visibilityOfElementLocated(By.name("description"))).sendKeys(text);
-        getDriver().findElement(By.name("Submit")).click();
-
-        TestUtils.returnToDashBoard(this);
-
-        getDriver().findElement(By.cssSelector("[href = 'job/" + PROJECT_NAME + "/']")).click();
-        getDriver().findElement(By.id("description-link")).click();
-        getDriver().findElement(By.name("description")).sendKeys(additionText);
-        getDriver().findElement(By.name("Submit")).click();
+        String DescriptionText = new HomePage(getDriver())
```
Comment (Lokankara) [inline src/test/java/school/redrover/MultiConfigurationProjectTest.java:67]: lowerCase naming

File: src/test/java/school/redrover/MultiConfigurationProjectTest.java [hunk 2]
```diff
@@ -59,26 +59,24 @@ public void testAddDescription() {
 
     @Test
     public void testEditDescriptionWithoutDelete() {
-        final String text = "qwerty123";
-        final String additionText = "AAA";
+        final String TEXT = "qwerty123";
```
Comment (Lokankara) [inline src/test/java/school/redrover/MultiConfigurationProjectTest.java:62]: there's no need to make it capsLock

File: src/test/java/school/redrover/MultiConfigurationProjectTest.java [hunk 3]
```diff
@@ -59,26 +59,24 @@ public void testAddDescription() {
...
-        getDriver().findElement(By.name("Submit")).click();
-
-        TestUtils.returnToDashBoard(this);
-
-        getDriver().findElement(By.cssSelector("[href = 'job/" + PROJECT_NAME + "/']")).click();
-        getDriver().findElement(By.id("description-link")).click();
-        getDriver().findElement(By.name("description")).sendKeys(additionText);
-        getDriver().findElement(By.name("Submit")).click();
+        String DescriptionText = new HomePage(getDriver())
+                .clickMCPName(PROJECT_NAME)
```
Comment (Lokankara) [inline src/test/java/school/redrover/MultiConfigurationProjectTest.java:68]: bad practice of using abbreviations in naming

**Review by Lokankara — APPROVED** [review]

**Pull request merged**